plugins/acousticbrainz_tonal-rhythm/acousticbrainz_tonal-rhythm.py

```diff
@@ -44,26 +45,38 @@
 class AcousticBrainz_Key:
 
     def get_data(self, album, track_metadata, trackXmlNode, releaseXmlNode):
+        if not musicbrainz_recordingid in track_metadata:            
+            log.error("%s: Error parsing response. No musicbrainz recording id found.",
```

Comment:
Capitalisation matters!

```suggestion
            log.error("%s: Error parsing response. No MusicBrainz recording id found.",
```

Reply:
Yeah, Sorry for error.